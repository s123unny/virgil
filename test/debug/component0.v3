component mytest0 {
	def main(arg: int) -> int {
		var x = mytest.new(arg, 2);
		var y = x.sum();
		return y;
	}
}

class mytest(a: int, b: int) {
	def sum() -> int {
		return a + b;
	}
}
