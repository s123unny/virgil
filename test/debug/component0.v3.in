start
bt
n
bt
s
bt
s
bt
s
q

