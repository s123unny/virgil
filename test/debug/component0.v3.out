stopOnStep
bt|mytest0.main|component0.v3|3
stopOnStep
bt|mytest0.main|component0.v3|4
stopOnStep
bt|mytest.sum|component0.v3|11
bt|mytest0.main|component0.v3|4
stopOnStep
bt|mytest0.main|component0.v3|5
end
result|2
